app/src/org/commcare/activities/CommCareHomeActivity.java

```diff
@@ -1433,76 +1438,16 @@ public void handlePullTaskResult(ResultAndError<DataPullTask.PullTaskResult> res
             return;
         }
 
-        DataPullTask.PullTaskResult result = resultAndErrorMessage.data;
-        String reportSyncLabel = result.getCorrespondingGoogleAnalyticsLabel();
-        int reportSyncValue = result.getCorrespondingGoogleAnalyticsValue();
-
-        switch (result) {
-            case AUTH_FAILED:
-                displayMessage(Localization.get("sync.fail.auth.loggedin"), true);
-                break;
-            case BAD_DATA:
-            case BAD_DATA_REQUIRES_INTERVENTION:
-                displayMessage(Localization.get("sync.fail.bad.data"), true);
-                break;
-            case DOWNLOAD_SUCCESS:
-                if (formsToSend) {
-                    reportSyncValue = GoogleAnalyticsFields.VALUE_WITH_SEND_FORMS;
-                } else {
-                    reportSyncValue = GoogleAnalyticsFields.VALUE_JUST_PULL_DATA;
-                }
-                displayMessage(Localization.get("sync.success.synced"));
-                break;
-            case SERVER_ERROR:
-                displayMessage(Localization.get("sync.fail.server.error"));
-                break;
-            case UNREACHABLE_HOST:
-                displayMessage(Localization.get("sync.fail.bad.network"), true);
-                break;
-            case CONNECTION_TIMEOUT:
-                displayMessage(Localization.get("sync.fail.timeout"), true);
-                break;
-            case UNKNOWN_FAILURE:
-                displayMessage(Localization.get("sync.fail.unknown"), true);
-                break;
-        }
-
-        if (userTriggeredSync) {
-            GoogleAnalyticsUtils.reportSyncAttempt(
-                    GoogleAnalyticsFields.ACTION_USER_SYNC_ATTEMPT,
-                    reportSyncLabel, reportSyncValue);
-        } else {
-            GoogleAnalyticsUtils.reportSyncAttempt(
-                    GoogleAnalyticsFields.ACTION_AUTO_SYNC_ATTEMPT,
-                    reportSyncLabel, reportSyncValue);
-        }
-        //TODO: What if the user info was updated?
```

Comment:
I don't really understand this TODO but suspect you may have fixed this issue in https://github.com/dimagi/commcare-odk/pull/814